app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -9,6 +9,7 @@ def self.ems_inv_to_hashes(inventory)
 
     def initialize
       @data = {}
+      @data[:container_image_registries] ||= []
```

Comment:
@enoodle for sure no need for the `||` you just defined it above. Maybe you can even do:

```ruby
@data = {
  # >>> Here you need a meaningful comment that explains this exception <<<
  :container_image_registries => []
}
```

Maybe you can even add a test... a refresh with no registries would it fail if it is `nil` instead of `[]` (if not we don't even need to initialize).